docs/staking-modules/csm/join-csm.md

```diff
@@ -0,0 +1,87 @@
+
+![join-csm-3](../../../static/img/csm/join-csm-3.png)
+
+There might be several bond curves (`getBondAmountByKeysCount` function implementations). A default curve is assigned to all Node Operators upon creation. A custom curve can be set for the Node Operator later.
+
+Existing Node Operators can top-up bond without uploading deposit data to compensate for the penalties or to have bond funds uploaded upfront.
+
+### Unbonded validators
+The term "unbonded" is introduced to refer to the validators for which the bond does not fully cover this validator. Taking into account the approach when the bond is common for all Node Operator's validators, unbonded validators can be determined in a way illustrated below. In the example, validator N+1 is unbonded.
+
+![join-csm-4](../../../static/img/csm/join-csm-4.png)
+
+### Possible negative stETH rebase consequences
+With the bond being stored in stETH, there is a risk of a reduction in the bond amount due to a negative stETH rebase. This might result in some Node Operators being unable to claim rewards (due to the actual bond being lower than required) or even validators becoming unbonded. This problem is described in detail in [Bond Mechanics in Lido ADR](https://hackmd.io/@lido/BJqWx7P0p). For this document, it is worth mentioning that no additional actions are required for CSM due to the low probability of the negative stETH rebase and a dedicated [fund](https://etherscan.io/address/0x8B3f33234ABD88493c0Cd28De33D583B70beDe35) at the Lido DAO's disposal for possible use as cover.
+
+## Deposit data validation and invalidation (aka vetting and unvetting)
+Given the upcoming [DSM](https://hackmd.io/@lido/rJrTnEc2a) upgrade, CSM will utilize an [optimistic vetting](https://hackmd.io/@lido/ryw2Qo5ia) approach. Uploaded deposit data will be treated as valid unless DSM reports it is not. In case of invalid deposit data detection, DSM calls [`decreaseVettedSigningKeysCount`](https://github.com/lidofinance/community-staking-module/blob/main/src/CSModule.sol#L861) to set `vettedKeys` pointer to the deposit data prior to the first invalid deposit data. In this case a Node Operator should remove invalid keys to resume stake allocation to the valid non-deposited keys.
+
+## Depositable keys
+Several factors determine if the deposit can be made using corresponding deposit data. This information is reflected in the Node Operator's `depositableKeys` property. This property indicates the number of deposit data records extracted sequentially starting from the last deposited record available in the Node Operator's key storage for deposits by the staking router. This number is determined as follows:
+-   `targetLimit` is not set -> `vettedKeys - depositedKeys - unbondedKeys`
+-   `targetLimit` is set -> `min(vettedKeys,targetLimit) - depositedKeys - unbondedKeys` 
+-   Node Operator has `stuckKeys != 0` no matter the `targetLimit` -> `0`.
```

Comment:
"Refunded" is unused in CSM. With the bond being less than 32 ETH and permissionless module entry, it makes no sense to "refund" a key. It can just be abandoned in the worst case